compiler/compile/OMRCompilation.hpp

```diff
@@ -969,15 +965,15 @@ class OMR_EXTENSIBLE Compilation
    void reportFailure(const char *reason);
 
 protected:
+   // Signature of the method we are compiling -- keep this at offset 0
```

Comment:
@mpirvu @klangman Would either of you happen to know why we kept signatures at offset 0?